src/python/pants/backend/python/lint/ruff/rules.py

```diff
@@ -64,7 +64,8 @@ async def run_ruff(
     )
 
     conf_args = [f"--config={ruff.config}"] if ruff.config else []
-    initial_args = ("--fix",) if request.is_fix else ()
+    # `--force-exclude` applies file excludes from config to files provided explicitly
+    initial_args = ("--force-exclude",) + (( "--fix",) if request.is_fix else ())
```

Comment:
This is the lint failure: 

```suggestion
    initial_args = ("--force-exclude",) + (("--fix",) if request.is_fix else ())
```